### PR TITLE
Merge ISA-Tab specification with ISA-XLSX changes

### PR DESCRIPTION
**Pull request by HLWeil**
Merged all current differences between ISA-Tab and ISA-XLSX as we use it into the base ISA-Tab specification.

Differences are listed here: #71 

This is no PR on the main specification but a preliminary PR on a side branch to be able to have a real discussion on the specification of ISA-Tab/ISA-XLSX differences.

@muehlhaus @Freymaurer @Brilator @kMutagene 

Please take a look and let me know about inconsistencies, miswordings or missing specification patterns.

### CODE REVIEW

File: ISA-XLSX.md [hunk 1]
```diff
@@ -1,19 +1,15 @@
-# ISA-Tab format
+# ISA-XLSX format
 
 For detail on ISA framework terminology, please read the [ISA Abstract Model specification](isamodel.md).
```
Comment (Brilator) [inline ISA-XLSX.md:3]: The linked md file does not exist

File: ISA-XLSX.md [hunk 2]
```diff
@@ -22,74 +18,47 @@ experiment; experimental steps (or sequences of events) are described in the Stu
...
-> - `STUDY PUBLICATIONS`
-> - `STUDY FACTORS`
-> - `STUDY ASSAYS`
-> - `STUDY PROTOCOLS`
-> - `STUDY CONTACTS`
+The purpose of top-level metadata sheets is aggregating and listing top-level metadata. Each sheet consists of sections consisting of a section header and key-value fields. Section headers MUST be completely written in upper case (e.g. STUDY), field headers MUST have the first letter of each word in upper case (e.g. Study Identifier); with the exception of the referencing label (REF).
 
 In the following sections, examples of each section block are given beside the specification of each section.
 
 For a full example of a complete Investigation File, please see [https://git.io/vD1va](https://git.io/vD1va).
```
Comment (Brilator) [inline ISA-XLSX.md:50]: this links to an ISA-Tab example that is txt not xlsx

File: ISA-XLSX.md [hunk 3]
```diff
@@ -22,74 +18,47 @@ experiment; experimental steps (or sequences of events) are described in the Stu
 Investigation file there may be one or more Studies defined with a corresponding Study file; for each Study there may
 be one or more Assays defined with corresponding Assay files.
```
Comment (Freymaurer) [inline ISA-XLSX.md:19]: Maybe specify that one assay can be referenced in multiple studies

File: ISA-XLSX.md [hunk 4]
```diff
@@ -164,19 +133,19 @@ This section MUST contain the following labels, with the specified datatypes for
...
+| Investigation Publication Status Term Accession Number | String or URI                                                                                      | The accession number from the Term Source associated with the selected term.                                                                                       |
 | Investigation Publication Status Term Source REF       | String                                                                                             | Identifies the controlled vocabulary or ontology that this term comes from. The Source REF has to match one the Term Source Name declared in the in the Ontology Source Reference section. |
 
-For example, the `INVESTIGATION PUBLICATIONS` section of an ISA-Tab `i_*.txt` file may look as follows:
+For example, the `INVESTIGATION PUBLICATIONS` section of an ISA-XLSX `isa.investigation.xlsx` file may look as follows:
 
 ```default
 INVESTIGATION PUBLICATIONS
```
Comment (Freymaurer) [inline ISA-XLSX.md:147]: maybe do these blocks as table. As different columns are very difficult to recognize in a code block.

File: ISA-XLSX.md [hunk 5]
```diff
@@ -442,34 +412,127 @@ Study Person Roles Term Accession Number	";;"	""	""	""	""	""	""
...
-follow the column cell in which the value is entered. For example, a characteristic type `Organism` with a value of `Homo sapiens`
-can be qualified with an `Ontology Annotation` of a term from NCBI Taxonomy as follows:
+Where a value is an `Ontology Annotation` in a table file, `Term Accession Number` and `Term Source REF` fields MUST follow the column cell in which the value is entered. These two columns SHOULD contain further ontological information about the header. In this case, following the static header string, separated by a sinlge space, there MUST be a short ontology term identifier formatted as CURIEs (prefixed identifiers) of the form `<IDSPACE>:<LOCALID>` (specified [here](http://obofoundry.org/id-policy)) inside `()` brackets.
+For example, a characteristic type `organism` with a value of `Homo sapiens` can be qualified with an `Ontology Annotation` of a term from NCBI Taxonomy as follows:
 
-| Characteristics[Organism]   | Term Source REF   | Term Accession Number                                |
+| Characteristics [organism]   | Term Source REF (OBI_0100026)  | Term Accession Number (OBI_0100026) |
```
[the remark this answers is not in the document]
Comment (HLWeil) [reply ISA-XLSX.md:531]: nope, copy error 👍

File: ARC specification.md [hunk 6]
```diff
@@ -127,7 +129,7 @@ Notes:
 
 ### ISA-XLSX Format
 
-ISA-XLSX follows the ISA model specification (v1.0) saved in a XLSX format. The XLSX format uses the SpreadsheetML markup language and schema to represent a spreadsheet document. Conceptually, using the terminology of the Spreadsheet ML specification [ISO/IEC 29500-1](https://www.loc.gov/preservation/digital/formats/fdd/fdd000398.shtml#:~:text=The%20XLSX%20format%20uses%20the,a%20rectangular%20grid%20of%20cells.), the document comprises one or more worksheets in a workbook. Every worksheet MUST contain one table object storing the metadata. Comments or auxillary information MAY be stored alongside with table objects in a worksheet.
+https://github.com/nfdi4plants/ARC-specfication/blob/main/ISA-XLSX.md
```
[the remark this answers is not in the document]
Comment (HLWeil) [reply ARC specification.md:132]: yet

File: ISA-XLSX.md (hunk 1 above)
Comment (HLWeil) [reply ISA-XLSX.md:3]: Naturally 👍

**Comment by kMutagene** [comment]
I think this is a nice first draft, one addition that i'd like would be internal links in the document. For example in the `Study File` section, there is this text block:

> The `Study File` MUST contain one `Top-Level Metadata sheet`. This sheet MUST be named `isa_study` and MUST contain the following sections:
>- `STUDY`
>- `STUDY DESIGN DESCRIPTORS`
>- `STUDY PUBLICATIONS`
>- `STUDY FACTORS`
>- `STUDY ASSAYS`
>- `STUDY PROTOCOLS`
>- `STUDY CONTACTS`

all of the mentioned sections should include a link to their respective specification.

File: ISA-XLSX.md (hunk 4 above)
Comment (kMutagene) [reply ISA-XLSX.md]: done, although i really do not like the examples from the initial ISA specs. the descriptions are far too large. I get where they come from, their sample should represent a 'real' investigation description. but that huge text block does really not help. I think @HLWeil will adapt the samples anyway.

File: ISA-XLSX.md [hunk 7]
```diff
@@ -1,22 +1,65 @@
...
+- [ISA-XLSX format](#isa-xlsx-format)
+- [Top-level metadata sheets](#top-level-metadata-sheets)
+  - [Ontology Source Reference section](#ontology-source-reference-section)
+    - [ONTOLOGY SOURCE REFERENCE](#ontology-source-reference)
+      - [Example](#example)
+  - [INVESTIGATION section](#investigation-section)
+    - [INVESTIGATION](#investigation)
+      - [Example](#example-1)
+    - [INVESTIGATION PUBLICATIONS](#investigation-publications)
+      - [Example](#example-2)
```
Comment (kMutagene) [inline ISA-XLSX.md:18]: I do not like how this looks too much, the idea was giving direct links to the samples, but i think this looks weird. other opinions?

Comment (HLWeil) [reply ISA-XLSX.md:18]: Maybe use `**Example**` instead of `#### Example`?

**Comment by HLWeil** [comment]
> I think this is a nice first draft, one addition that i'd like would be internal links in the document. For example in the `Study File` section, there is this text block:
> 
> > The `Study File` MUST contain one `Top-Level Metadata sheet`. This sheet MUST be named `isa_study` and MUST contain the following sections:
> > 
> > * `STUDY`
> > * `STUDY DESIGN DESCRIPTORS`
> > * `STUDY PUBLICATIONS`
> > * `STUDY FACTORS`
> > * `STUDY ASSAYS`
> > * `STUDY PROTOCOLS`
> > * `STUDY CONTACTS`
> 
> all of the mentioned sections should include a link to their respective specification.

YES! I think this will make navigating the file much easier. Also for stuff like "Annotation Table sheet" and "Ontology Annotation"

**Comment by kMutagene** [comment]
@HLWeil do you still intend to adapt the examples (e.g. shorter descriptions, other names, etc.)? Otherwise LGTM as far as i can tell

**Comment by HLWeil** [comment]
> @HLWeil do you still intend to adapt the examples (e.g. shorter descriptions, other names, etc.)? Otherwise LGTM as far as i can tell

Yes I'm still planning to 😄

**Pull request merged**